app/src/main/java/au/com/shiftyjelly/pocketcasts/ui/MainActivity.kt

```diff
@@ -499,8 +497,7 @@ class MainActivity :
             var showDialog by rememberSaveable { mutableStateOf(false) }
             if (showDialog) {
                 StoriesPage(
-                    viewModel = storiesViewModel,
-                    showDialog = showDialog,
+                    showDialog = true,
```

Comment:
Do we still need this`showDialog` parameter? It looks like we're always passing `true` now, but maybe that's just temporary?

Reply:
Not needed anymore, removed in 0ac80aa.